File: Selenium/Project/Project4Test.java
package ProjectJob;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Project4Test {
    WebDriver driver;
    
    // Setup function
    @BeforeClass
    public void setUp() {
        // Initialize Fire fox driver
        driver = new FirefoxDriver();
        // Open the page
        driver.get("https://alchemy.hguy.co/jobs");
    }
    
    // Test function
    @Test(priority = 1)
    public void homePageTest() {
    	
    	// Locate the second heading
    	WebElement heading = driver.findElement(By.xpath("//h2")); 
    	String secondHeading = heading.getText();
    	System.out.println("Second Heading: " + secondHeading);
        
        // Assert page title
        Assert.assertEquals(secondHeading, "Quia quis non");
        
        
    }
    
    
    // Tear down function
    @AfterClass
    public void tearDown() {
        // Close the browser
        driver.quit();
    }
}

File: Selenium/Project/Project4test.py

from selenium import webdriver 
from selenium.webdriver.common.by import By
import pytest 


@pytest.fixture 
def browser():
    # Initialize the Chrome WebDriver
    driver = webdriver.Chrome()
    yield driver
    # Quit the WebDriver
    driver.quit()

def test_verify_secondheading(browser):
        browser.get("https://alchemy.hguy.co/jobs")

        # Print the title of the page
        
        actual_second_heading = browser.find_element(By.XPATH,"//h2")
        actual_second_heading_verify = actual_second_heading.text

        
        # Verify the title 
        assert actual_second_heading_verify == 'Quia quis non', f"Expected title to be 'Quia quis non', but got '{actual_second_heading_verify}'"
        print("The Actual second heading is:", actual_second_heading_verify)
if __name__ == '__main__':
    pytest.main()
